fex2/preconfigured/experiments/common.sh

```diff
@@ -14,37 +14,62 @@ trap "trap - SIGTERM && set +e; kill -- -$$ 2>/dev/null" SIGINT SIGTERM
 trap "exit_code=\$?; set +e; kill -- -$$ 2>/dev/null ; exit \$exit_code" EXIT
 
 # Logging and errors
+DEBUG=''
+WARNING=''
+ERROR=''
+ENDC=''
+
+# TODO: Remove function keyword as it's deprecated
 function debug() {
     echo -e "${DEBUG}[DEBUG]" "$@" "$ENDC"
 }
 
 function error_exit() {
-    echo -e "${ERROR}[ERROR]" "$1" "$ENDC"
+    echo -e "${ERROR}[ERROR] $1 $ENDC"
+    exit "$2"
+}
+
+function unexpected_error_exit() {
```

Comment:
The problem I had was, that:
- In case of for instance a missing variable, I want to output an error incl. the stack.
- In case of for instance a missing input for an experiment, I want to output just an error.
We could also either always output the stack or never :thinking:

Reply:
Ok, got it. Then rename it to something like `error_exit_with_backtrace`